app/(entrance)/reset-password/components/ResetPasswordPage.js

```diff
@@ -59,11 +58,11 @@ export default function ResetPasswordPage({ email, token }) {
     if (isValid) {
       const res = await resetPassword(email, password, token);
 
-      if (res) {
+      if (res.ok) {
         setResetSuccessful(true);
         successSnackbar(`Your password has been updated`);
       } else {
-        errorSnackbar(`Error updating password`);
+        errorSnackbar(`Error updating password: ${res.data?.message}`);
```

Comment:
Is there a chance this message could contain `Not found` or something else that could result in a way for a bad actor to perform "Credential Stuffing" or "Forgot Password Enumeration Attack"?